tm2/pkg/amino/genproto/snakecase.go

```diff
@@ -0,0 +1,335 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file has ToLowerSnakeCase which the Protobuf linter uses to make
+// the expected spelling of a message field name. It is a copy of:
+// https://github.com/bufbuild/buf/blob/main/private/pkg/stringutil/stringutil.go
+
+package genproto
+
+import (
+	"sort"
+	"strings"
+	"unicode"
+)
+
+// TrimLines splits the output into individual lines and trims the spaces from each line.
+//
+// This also trims the start and end spaces from the original output.
+func TrimLines(output string) string {
```

Comment:
Please either unexport all those functions or relocate them to a subfolder such as `genproto/stringutils`. This will help maintain the focus of the Go API for `genproto` on generating protofiles. Thank you.

Reply:
I pushed a commit to move them to a subfolder.